### ISSUE TITLE
Track Energy Levels & Current Thoughts for Tasks

### ISSUE THREAD

**Issue by krishnakanthb13** [Razeeman/Android-SimpleTimeTracker#251]
**Feature Request: Track Energy Levels & Current Thoughts for Tasks**

**Description:**
Add an optional feature to track energy levels and thoughts both *before* and *after* completing a task or event. This feature would prompt users to input their energy level and a short note/comment on their current thoughts, allowing them to reflect on how they feel going into a task and how they feel once it's complete. 

**Key Details:**
- **Energy Level Tracking:** Enable users to select an energy level (e.g., "High," "Moderate," "Low") both before starting a task and after completing it. This will help users understand how different tasks impact their energy.
- **Current Thought Commenting:** Provide a simple text field that pops up when moving to a new task or event, where users can jot down their current thoughts or reflections both before and after the task.
  
**Settings:**
- These features should be optional and configurable in the app's settings so that users can enable or disable them on demand.
- Both of the above options can be prompted together based on the app's settings.

**Rationale:** These features would allow users to better track their energy and mindset in relation to their activities, enabling more mindful task planning and energy management.

**Comment by cogk** [comment]
related to https://github.com/Razeeman/Android-SimpleTimeTracker/issues/143
